### PR TITLE
Accelerator WIP

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

File: configs/secrets/dev-genesis-secrets.js [hunk 1]
```diff
@@ -8,11 +8,22 @@ const path = require('path');
...
 config['veres-one'].maintainerConfigFile =
   path.join(config.paths.secrets, 'maintainer.ddo');
-config['veres-one'].governorsConfigFile =
-  path.join(config.paths.secrets, 'governors.ddo');
 config['veres-one'].maintainerPassphrase =
   'insecure_eyium0phookoh9geeshewomaekuoTeib';
+
+// governors
+config['veres-one'].governorsConfigFile =
+  path.join(config.paths.secrets, 'governors.ddo');
```
Comment (dlongley) [inline configs/secrets/dev-genesis-secrets.js]: `ddo` isn't a thing anymore, probably shouldn't name it that. Perhaps just `.json` or `.jsonld`?

File: lib/accelerator.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
+/*!
+ * Veres One Accelerator API.
+ *
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+const _ = require('lodash');
```
Comment (dlongley) [inline lib/accelerator.js]: `use strict`

File: lib/accelerator.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+bedrock.events.on('bedrock-express.ready', (app, callback) =>
+  _setupAccelerator(callback));
+
+// try to read and process, or generate save and process
+function _readOrCreateJson({path, create, loaded, stored}, callback) {
+  // try to read
+  fs.readFile(path, 'utf8', (err, data) => {
+    if(err) {
+      if(err.code === 'ENOENT') {
```
Comment (dlongley) [inline lib/accelerator.js]: Code doesn't have to indented so much if you bail early on `!(err && err.code === 'ENOENT')`.

File: lib/accelerator.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+// module API
+const api = {};
+module.exports = api;
+
+bedrock.events.on('bedrock-express.ready', (app, callback) =>
+  _setupAccelerator(callback));
+
+// try to read and process, or generate save and process
+function _readOrCreateJson({path, create, loaded, stored}, callback) {
```
Comment (dlongley) [inline lib/accelerator.js]: Seems like this is a utility function that might find a better home in a lower-level lib like `did-veres-one` if nowhere else.

File: lib/accelerator.js (hunk 3 above)
Comment (dlongley) [reply lib/accelerator.js]: Can do a TODO for that, doesn't need to be done right away.

File: configs/secrets/dev-genesis-secrets.js (hunk 1 above)
Comment (davidlehn) [reply configs/secrets/dev-genesis-secrets.js]: Changed.

File: lib/accelerator.js (hunk 2 above)
Comment (davidlehn) [reply lib/accelerator.js]: Added.

File: lib/accelerator.js (hunk 4 above)
Comment (davidlehn) [reply lib/accelerator.js]: Yes, I'd rather this be somewhere else.  Can revisit.

**Pull request merged**